platform-includes/getting-started-install/javascript.angular.mdx

```diff
@@ -1,37 +1,35 @@
 ```bash {tabTitle:npm}
-# Angular 12 and newer:
-npm install --save @sentry/angular-ivy
-
-# Angular 10 and 11:
 npm install --save @sentry/angular
 ```
 
 ```bash {tabTitle:Yarn}
-# Angular 12 and newer:
-yarn add @sentry/angular-ivy
-
-# Angular 10 and 11:
 yarn add @sentry/angular
 ```
 
 ### Angular Version Compatibility
 
-Because of the way Angular libraries are compiled, you need to use a specific version of the Sentry SDK for each corresponding version of Angular as shown below:
+The Sentry Angular SDK supports Angular 14 and newer in its current major version.
+
+If you're using an older version of Angular, you also need to use an older version of the SDK:
 
-| Angular version | Recommended Sentry SDK |
-| --------------- | ---------------------- |
-| 12 and newer    | `@sentry/angular-ivy`  |
-| 10, 11          | `@sentry/angular`      |
-| Older versions  | See note below         |
+| Angular version | Recommended Sentry SDK                                            |
+| --------------- | ----------------------------------------------------------------- |
+| 14 and newer    | `@sentry/angular`                                                 |
+| 12 or 13        | `@sentry/angular-ivy@^7` (see [Note](#what-is-sentryangular-ivy)) |
+| 10 or 11        | `@sentry/angular@^7`                                              |
 
 <Alert level="warning">
 
-Support for any Angular version below 10 was discontinued in version 7 of the SDK. If you need to use Angular 9 or older, try version 6 (`@sentry/angular@^6.x`) of the SDK. For AngularJS/1.x, use `@sentry/browser@^6.x` and our [AngularJS integration](/platforms/javascript/guides/angular/angular1). Note, that these versions of the SDK are no longer maintained or tested.
+Support for Angular versions below 14 was discontinued in version 8 of the SDK.
+If you need to use Angular 13 or older, you can continue using version 7 of the SDK (see table above).
+If you need to use Angular 9 or older, you can continue using version 6 of the SDK.
+For AngularJS/1.x, use `@sentry/browser@^6` and our [AngularJS integration](/platforms/javascript/guides/angular/angular1).
+Please note, that these versions of the SDK are no longer maintained or tested.
```

Comment:
```suggestion
* These versions of the SDK are no longer maintained or tested.
```

Can we remove the entire alert (lines 21-30) and add the extra info to the table itself? I think this alert is pretty long to read and can easily be summarized in a table anyway. For the versions of the SDK that are no longer maintained, let's add a star (*) to it in the table.